scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -22,14 +22,14 @@ type LaneOptions = {
 
 export class LaneListCmd implements Command {
   name = 'list';
-  description = `list lanes`;
+  description = `list locally imported/created lanes`;
   alias = '';
   options = [
     ['d', 'details', 'show more details on the state of each component in each lane'],
-    ['j', 'json', 'show lanes details in a json format'],
-    ['r', 'remote <remote-scope-name>', 'show remote lanes'],
-    ['', 'merged', 'show merged lanes'],
-    ['', 'not-merged', 'show lanes that are not merged'],
+    ['j', 'json', "show lanes' details in a json format"],
+    ['r', 'remote <remote-scope-name>', 'show remote lanes'], // @david does this show all lanes in the whole organisation?
```

Comment:
bit has no idea what is an "org". it only works with scopes, so this shows all lanes in a remote scope.

Reply:
then i'll update the description